doc/spec_exceptionHandling.dox

```diff
@@ -43,7 +46,7 @@ namespace ChimeraTK {
   - \anchor exceptionHandling_b_2_1 2.1 The exception status is published as a process variable together with an error message. [\ref testExceptionHandling_b_2_1 "T"]
     - 2.1.1 The variable \c Devices/\<alias\>/status contains a boolean flag whether the device is in an error state.
     - 2.1.2 The variable \c Devices/\<alias\>/message contains an error message, if the device is in an error state, or an empty string otherwise.
-  - \anchor b_2_2 2.2 Read operations will propagate the DataValidity::faulty flag to the owning module / fan out (without changing the actual data value of the process variable):
+  - \anchor b_2_2 2.2 Read operations will propagate the DataValidity::faulty flag to the owning module / fan out:
```

Comment:
Is it more appropriate for the data validity propagation spec to test this?

Reply:
Yes, the "propagation" part is tested there. However, that spec is very brief.This point is also about the ExceptionHandlingDecorator changing the flag. And this needs to be tested. The data validity propagation spec refers to this spec about it and does not specify anything where the changing flags are coming from. Maybe we there refer to this test. From the context it fit better here.